geomesa-spark/geomesa-spark-jts/src/main/scala/org/locationtech/geomesa/spark/jts/util/RowGeoJSON.scala

```diff
@@ -0,0 +1,75 @@
+/***********************************************************************
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.spark.jts.util
+
+import java.{lang => jl}
+
+import com.vividsolutions.jts.geom.Geometry
+import com.vividsolutions.jts.io.geojson.GeoJsonWriter
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.jts.JTSTypes
+import org.apache.spark.sql.types._
+
+class RowGeoJSON(structType: StructType, var geomOrdinal: Option[Int] = None) {
```

Comment:
To be picky here, I'd put `setGeometryOrdinal()` into a `RowGeoJSON` companion object and call the function as the default. e.g. 
```
import RowGeoJSON.defaultGeometryOrdinal
class RowGeoJSON(structType: StructType, geomOrdinal: Int = defaultGeometryOrdinal) {...}
object RowGeoJSON { 
  def defaultGeometryOrdinal: Int = ...
}
```
I see calling option.get later as a bad thing in general. But that may just be personal preference.

Reply:
Yeah I felt like there was a better pattern out there when writing this. Was kinda hoping someone would point one out ;)

Reply:
yeah, it's probably better to move the logic in `setGeometryOrdinal` into an `apply` method in a companion object. you could then remove the default value for geomOrdinal.

Reply:
basically what @aheyne said, but since you need to pass in the structType you won't be able to use it as a default value (unless you create a secondary argument list, but eh)